src/collections/blog/2023/2023-08-07-layer5-badge-program/index.mdx

```diff
@@ -0,0 +1,194 @@
+
+Every aspect of what we do revolves around our contributors. We are excited to introduce Layer5 Badges, a way for your outstanding contributions to be recognized and rewarded!
+
+### What Are Layer5 Badges?
+
+These badges are a representation of your contributions and milestones that you achieve together with the Layer5 community. This is how contributors demonstrate their ownership, dedication, skills and loyalty to Layer5. There are several badges each representing a specific contribution and a skillset.
+
+It does not matter if you are an experienced developer, a newbie figuring out your way into open source, documentation enthusiast, a community socialite... we have a badge you can earn and display proudly!
+
+### Why Are Badges Important?
+
+  - **Recognition**: Badges provide tangible recognition for your hard work and commitment. They showcase your achievements and highlight the areas where you've made a positive impact.
+  - **Motivation**: Striving to earn badges can serve as a powerful motivator for every contributor. As you earn badges, you'll gain a sense of accomplishment, pushing you to set new goals and bring forth new ideas for the betterment of the Layer5 community.
+  - **Community Building**: Badges help foster a sense of belonging among contributors. They encourage collaboration, healthy competition, and the sharing of knowledge and experience. We recognize such individuals as [MeshMates](/community/meshmates).
+  - **Resume Boost**: Your Layer5 badges can help you stand out in your resume, LinkedIn profile, and other professional platforms. They serve as concrete evidence of your expertise and your dedication to open source hence helping your personal growth.
+ - **Social Media Showcase**: These badges can be shared on several social media platforms! Let your friends, family, colleagues and your bosses know about your contributions, your journey, and your passion for cloud native technology.
+
+<img src={Playfield} />
+
+### How Can You Earn Badges?
+
+Layer5 badges are divided into two categories, namely <i>Project Badges</i> and <i>Achievement Badges</i>.
+
+<i>Project Badges</i> are assigned by the admin to the user when a user contributes to a particular project. Achievement Badges are the badges that are assigned automatically when a user completes a task like sharing a design, creating a design for the first time, etc.
+
+### Can I Share My Badge On My Profile
+
+Use the badge embed codes.
+
+### Your Layer5 Profile
+
+https://meshery.layer5.io/users
+
+### Your Layer5 Community Member Card
+
+https://layer5.io/community/members
+
+<h2 align="center">Layer5 Badges</h2>
+
+<h3>Achievement Badges</h3>
+<div class="badge-grid">
+<div>
+  <img src="https://raw.githubusercontent.com/layer5labs/meshery-extensions-packages/master/assets/badges/bring-a-buddy/bring-a-buddy.svg" /><span>Bring a Buddy</span> <p>This badge is awarded to the users who invite someone to Layer5 cloud.</p>
```

Comment:
@thisiskaransgit or @gopivaibhav Will you please replace this and other badge raw links with https://badges.layer5.io/assets/badges/bring-a-buddy/bring-a-buddy.svg according to badge?

Reply:
This is the one instance in which we'll be using the PNGs.

Reply:
Actually, this needs to be converted to read from files / data file.

Reply:
Okay. :ok_hand: